client/src/components/Userpage.js

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Grid, Button } from "@material-ui/core";
+import Avatar from "./BigAvatar";
+import ExploreCard from "./ExploreCard";
+import "./userpage.css";
+
+const useStyles = makeStyles((themes) => ({
+  grid: {
+    width: "100%",
+    margin: "0px",
+  },
+  leftQuadrant: {
+    width: "30%",
+    margin: "0px",
+  },
+  rightQuadrant: {
+    width: "90%",
+    margin: "0px",
+    marginLeft: "30%",
+    backgroundImage:
+      "linear-gradient(to right, rgba(240,240,248,1), rgba(240,240,248,0));",
+    position: "absolute",
+    marginRight: "-30%",
+  },
+  paper: {
+    textAlign: "center",
+    backgroundColor: "transparent",
+  },
+  avatar: {
+    marginLeft: "41%",
+  },
+  container: {
+    display: "grid",
+  },
+  exploreContainer: {
+    marginLeft: "13%",
+    marginRight: "-13%",
+  },
+  invisible: {
+    marginLeft: "10%",
+    marginRight: "-10%",
+    opacity: "0",
+  },
+  alignment: {
+    alignContent: "center",
+    textAlign: "center",
+    margin: "auto",
+  },
+  logout: {
+    position: "absolute",
+    botton: "10px",
+    alignContent: "center",
+    textAlign: "center",
+    margin: "auto",
+  },
+  selected: {
+    textAlign: "center",
+    color: "#F09600",
+    marginLeft: "-38%",
+  },
+}));
+
+export default function Userpage2() {
+  const classes = useStyles();
+
+  return (
+    <Grid container spacing={0} className={classes.grid}>
+      <Grid container spacing={1} className={classes.leftQuadrant}>
+        <Grid item xs={12} />
+        <Grid item xs={12} />
+        <Grid item xs={12} />
+        <Grid item xs={12} />
+        <Grid item xs={12} />
+        <Grid item xs={12} className={classes.container}>
+          <paper className={classes.alignment}>
+            <Avatar />
+          </paper>
+        </Grid>
+        <Grid item xs={12}>
+          <h3 className={classes.alignment}>John Doe</h3>
+          <p className={classes.alignment} style={{ color: "#9EA1BC" }}>
```

Comment:
To be more consistent and use Material UI components, use `Typography` for all texts. By default it acts as a `p`, and you can use the `variant` prop to make it a header